src/organisms/cards/PolicyCard/propTypes.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+
+export default {
+  /**
+   * This prop will add a new className to any inherent classNames
+   * provided in the component's index.js file.
+   */
+  className: PropTypes.string,
+  /**
+   * The insurance carrier logo
+   */
+  carrierLogo: PropTypes.object.isRequired,
+  /**
+   * The type of policy
+   */
+  policyType: PropTypes.string.isRequired,
+  /**
+   * More information about the policy
+   */
+  policyTooltip: PropTypes.node.isRequired,
+  /**
+   * The financial strength of the carrier
+   */
+  financialStrength: PropTypes.string.isRequired,
+  /**
+   * The customer service rating of the carrier
+   */
+  customerService: PropTypes.string.isRequired,
+  /**
+   * The total number of customers for the carrier
+   */
+  totalCustomers: PropTypes.number.isRequired,
+  /**
+   * The insurance carrier logo
+   */
+  monthlyPremium: PropTypes.number.isRequired,
+  /**
+   * Optional discount percentage
+   */
+  discount: PropTypes.node,
+  /**
+   * Optional array of nodes that will be spaced appropriately in the footer
+   */
+  footer: PropTypes.array,
```

Comment:
my thinking here (and for `policyTooltip`) is that the icon will have to be styled and aligned during the implementation. This is because all icons are slightly different, and there's not one consistent way to change the color of a svg or set the vertical alignment

Reply:
@cisacke sounds good to me